Practical-11/Practical-11/Views/Test02/_Create.cshtml

```diff
@@ -0,0 +1,52 @@
+﻿@using Practical_11.Models
+@model Practical_11.Models.User
+
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Create User</title>
+</head>
+<body>
+    <h1>Craete User </h1>
```

Comment:
spelling error

Reply:
record gets saved without any input which is not correct proper validations are needed